doc/classes/PhysicsServer3D.xml

```diff
@@ -1014,10 +1014,261 @@
+			</description>
+		</method>
+		<method name="soft_body_get_simulation_precision" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the simulation precision of a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_space" qualifiers="const">
+			<return type="RID" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the [RID] of the space assigned to a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_state" qualifiers="const">
+			<return type="Variant" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="state" type="int" enum="PhysicsServer3D.BodyState" />
+			<description>
+				Returns a soft body state (see [enum BodyState] constants).
+				[b]Note:[/b] Godot Physics does not support [constant BODY_STATE_LINEAR_VELOCITY], [constant BODY_STATE_ANGULAR_VELOCITY], [constant BODY_STATE_SLEEPING], or [constant BODY_STATE_CAN_SLEEP].
+			</description>
+		</method>
+		<method name="soft_body_get_total_mass" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the total mass assigned to a soft body.
+			</description>
+		</method>
+		<method name="soft_body_is_point_pinned" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="point_index" type="int" />
+			<description>
+				Returns whether a soft body point at the specified index is pinned.
+			</description>
+		</method>
+		<method name="soft_body_move_point">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="point_index" type="int" />
+			<param index="2" name="global_position" type="Vector3" />
+			<description>
+				Moves a soft body point at the specified index to a position in global coordinates.
+			</description>
+		</method>
+		<method name="soft_body_pin_point">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="point_index" type="int" />
+			<param index="2" name="pin" type="bool" />
+			<description>
+				Pins or unpins a soft body point at the specified index based on the value of [param pin].
+				[b]Note:[/b] Pinning a point effectively makes it kinematic, preventing it from being affected by forces, but you can still move it using [method soft_body_move_point].
+			</description>
+		</method>
+		<method name="soft_body_remove_all_pinned_points">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Unpins all points of a soft body.
+			</description>
+		</method>
+		<method name="soft_body_remove_collision_exception">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="body_b" type="RID" />
+			<description>
+				Removes a body from the list of bodies exempt from collisions.
```

Comment:
This is copied straight from `body_remove_collision_exception` but it doesn't have to be.
```suggestion
				Removes a soft body from the list of bodies exempt from collisions.
```

Reply:
Woops, good catch, that was not my intention.

Reply:
Oh wait, this one isn't actually incorrect, since "body" in this case refers to all kinds of bodies, not only soft bodies.